### DISCUSSION TITLE
Hide stray dogs specifically in Entity Hider

### DISCUSSION THREAD

**Discussion by Voileexperiments** [runelite/runelite#18480] [Ideas]
Some stray dog NPCs in the game are detrimental to skilling, like Molossus in Varlamore plaza which blocks mouse clicks on wealthy citizens when afk thieving.

Currently stray dogs can only be hidden with "hide attackers" on, but it only works when the dog is targeting you, not to mention it only happens to work *incidentally* and so isn't a working solution (for example, they aren't hidden this way if the dog is target someone else, or being shooed away atm).

**Comment by DominickCobb-rs** [comment]
no.

Jokes aside, just menu-entry swap the left click to be walk here for the stray dogs.
